aQute.libg/bnd.bnd

```diff
@@ -9,8 +9,9 @@
 	osgi.core;version=latest
 
 -testpath: \
-	${junit},\
-	osgi.core;version=latest,\
+	${junit-platform},\
```

Comment:
We want to have the `${junit}` macro represent the same jars for all projects to use for their `-testpath`. I don't want to have each project pick its own adventure here. This is why `${junit}` currently includes AssertJ. So I think we need a value for `${junit}` which all bundles will use.

Change this will require the `-buildpath` for `biz.aQute.junit` be changed to replace its use of `${junit}` with the current value of `${junit}`.